src/NUnitEngine/nunit.engine/Internal/NUnitConfiguration.cs

```diff
@@ -59,35 +59,28 @@ public static string MonoExePath
         {
             get
```

Comment:
In this method, searchNames was unused. It seems to relate to old functionality to look for mono on the path, which was previously removed, but this var left. Also reduced the nesting of if's a little here - shoulnd't be any difference in functionality.